src/services/section-helper.js

```diff
@@ -99,3 +99,20 @@ export function createSectionRevision({ section, ancestorSection = null, isResto
   logger.info(`Creating new revision for section key ${section.key}`);
   return createNewSectionRevisionFrom(section);
 }
+
+export function extractCdnResources(sections, pluginInfoFactory) {
+  return [
+    ...sections.reduce((cdnResources, section) => {
+      const info = pluginInfoFactory.tryCreateInfo(section.type);
+      if (info && section.content) {
+        info.getCdnResources(section.content)
+          .forEach(resource => {
+            if (resource) {
+              cdnResources.add(resource);
+            }
+          });
+      }
+      return cdnResources;
```

Comment:
I think this would work and make it easier to follow here 🤔 
```
return info && section.content
    ? [ ...cdnResources, ...info.getCdnResources(section.content).filter(resource => resource)]
    : cdnResources;
```

Reply:
Yes, that's much easier to read, thanks for the suggestion!